models/__init__.py

```diff
@@ -4,20 +4,20 @@
 from time import gmtime, strftime
 
 def _get_all_piskels_for_user(user_id, limit=20):
-  mem_key = "user_piskels_" + str(user_id)
+  mem_key = 'user_piskels_' + str(user_id)

```

Comment:
That's fine, since it's a deliberate change. You did isolate it in a commit. but Github's review UI makes it difficult to see that.

Thanks for making this consistent in the codebase!